### ISSUE TITLE
Not booting into PSBBN menu

### ISSUE THREAD

**Issue by survive9** [CosmicScale/PSBBN-Definitive-English-Patch#24]
Hi, I hate to use the issue tracker for help on this program but I went and installed PSBBN on my 1 TB HDD and added a couple of games to test. When I inserted the HDD into the PS2, the console didnt load into the PSBBN menu but instead went to the regular memory card/disc menu on boot. I am using an official network adapter which I modified with Bitfunx's IDE to SATA addon and the HDD worked with FHDB previously. These are the logs

[PSBBN-installer.log](https://github.com/user-attachments/files/18236716/PSBBN-installer.log)
[game-installer.log](https://github.com/user-attachments/files/18236769/game-installer.log)

**Comment by CosmicScale** [comment]
I just took a look at the logs and I can't see any problems with the install. It looks like a hardware problem to me and your PS2 isn't registering that a HDD is inserted.

Try launching LaunchELF on your console, go to MISC > HddManager. At the top if it says "CONNECTED: NO" then it is definitely a hardware issue.

**Comment by survive9** [comment]
It does read the HDD and all the partitions in APA
![IMG_1780](https://github.com/user-attachments/assets/84b00436-e258-421d-aab0-c600afecf60f)
Is it possible it’s a timeout issue where the HDD doesn’t spin up fast enough on boot?

**Comment by CosmicScale** [comment]
The PSBBN install competed without an error, the PS2 is picking up the drive and FreeHDBoot previously worked... I'm clueless.

**Comment by survive9** [comment]
Welp, the hard drive is an HP model MM1000GBKAL. It’s a 7200 RPM HDD for future reference. I guess for whatever reason PSBBN doesn’t like it so avoid it

**Comment by CosmicScale** [comment]
> I guess for whatever reason PSBBN doesn’t like it so avoid it

This doesn’t sound like an incompatible drive, as others have reported those typically still launch into PSBBN. If possible, try using a different drive or a direct SATA connection instead of USB to see if that resolves the issue.

**Comment by survive9** [comment]
> This doesn’t sound like an incompatible drive, as others have reported those typically still launch into PSBBN. If possible, try using a different drive or a direct SATA connection instead of USB to see if that resolves the issue.

This is direct SATA not usb, I’ll try a different drive though, maybe a SSD that supports garbage collection and has DRAM like the Crucial MX300.

**Comment by CosmicScale** [comment]
Are you using the Official Sony Network Adapter? Third-party adapters are not compatible with PSBBN

**Comment by survive9** [comment]
Yes I’m using the official network adapter with bitfunx’s sata adapter

**Comment by kantosCoder** [comment]
Hello, similar problem here. Official HDD adaptor, kingston s400 950gb (the one you recommend). The first time after installing boots normally into PSBBN, configured network, all ok, launching a game, all ok. The problem comes when i reboot the console, it never boots again into PSBBN. Tried to install it again, as before, works only in the first boot.  Memory cards formated again, everithing clean... Dunno what can it be. Using another ssd with the old opl everything works normal so I assume it-s not the adaptor. My guess is maybe a cfg file of some sort. I dont know what to do. Thanks in adavance!

**Comment by CosmicScale** [comment]
> it never boots again into PSBBN.

How so exactly? Does it freeze at the 'PlayStation 2' logo while loading PSBBN, or does it go to the regular PS2 browser? Can the drive still be read by wLaunchELF? How did you turn off your console?

**Comment by kantosCoder** [comment]
> > it never boots again into PSBBN.
> 
> How so exactly? Does it freeze at the 'PlayStation 2' logo while loading PSBBN, or does it go to the regular PS2 browser? Can the drive still be read by wLaunchELF? How did you turn off your console?

It does the "im reading the hdd" thing (slows the startup) but never manages to boot the psbbn and goes to the standard ps2 dash. I did turn off the ps2 normally, by the front power off button. The first time I tried it was with 200 Game partitions and like 20-30 gb for music, so the second time I tried with only 150 game partitions (with 125 games using then) just in case, but the samme happens. First boot: flawless, second boot: no more psbbn.

**Comment by kantosCoder** [comment]
I'd also add that my clock battery is dead, but the two times I had set the clock correctly just in case. ¿Could it have anything to do with it?

Edit: also, my ps2 fat is the silver model with no ilink port, and the two times i did connect my ps2 to the Internet (without trouble in that regard)

Edit 2: I'm also Pal Spain, connected via RGB to a sony trinitron which supports both ntsc and pal signals (I know that maybe this info doesn't matter and sorry for that many edits, I'm trying to give you the more context posible) 

Edit 3: Psbbn didn't write any file in the memcards after boot (dont know if it should, just more context)

**Comment by CosmicScale** [comment]
The boot sector on the drive must be corrupt but I have no idea why that would happen, especially after a successful boot. It all points to a hardware problem. All I can suggest is trying a different drive or a different SATA mod on your console. I use the Kaico IDE to SATA Upgrade Kit.

**Comment by CosmicScale** [comment]
Try downloading osdboot.elf from [here](https://archive.org/download/osdboot) and run it on your PS2. Does PSBBN boot?

**Comment by survive9** [comment]
So I replaced the original hard drive with another 1 TB drive and it now booted into the menu. I wonder if the original being a 7200 rpm drive messed something up since this one’s a 5400 rpm

**Comment by CosmicScale** [comment]
> So I replaced the original hard drive with another 1 TB drive and it now booted into the menu. I wonder if the original being a 7200 rpm drive messed something up since this one’s a 5400 rpm

Glad you got it working. I'm not sure what the cause is but trying a different drive is usually the solution.
